helm/kong-app/README.md

```diff
@@ -339,6 +339,9 @@ helm install stable/kong --set ingressController.enabled=true \
   --set postgresql.enabled=false --set env.database=off
 # with a database
 helm install stable/kong --set ingressController.enabled=true
+# with a external database
```

Comment:
```suggestion
# with an external database
```

I love the docs get extended but should it also include the postgres host and password?

Reply:
true I will do